### PR TITLE
adding k8s 1.8 to ci pipeline

### LINKED ISSUE #1032: Add coverage for k8s 1.8 to the build pipeline
We need to run integration tests on k8s 1.8

### PR DESCRIPTION
**Pull request by dangula**
fix #1032 
[skip ci]- just added comment to kubeadm script.

### CODE REVIEW

File: tests/scripts/kubeadm-install.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ KUBE_VERSION=${1:-"v1.7.5"}
 
 null_str=
 KUBE_INSTALL_VERSION="${KUBE_VERSION/v/$null_str}"-00
-
+sudo swapoff -a
```
Comment (travisn) [inline tests/scripts/kubeadm-install.sh]: why do we need to disable swap? a comment would be good here

File: Jenkinsfile [hunk 2]
```diff
@@ -67,7 +67,8 @@ pipeline {
                 script{
                     def data = [
                         "aws_ci": "v1.6.7",
-                        "gce_ci": "v1.7.5"
+                        "gce_ci": "v1.7.5",
+                        "gce_ci_1.8": "v1.8.0"
```
Comment (travisn) [inline Jenkinsfile]: could we rename all these so it is obvious in the jenkins ui which version is running for each platform? For example, what about `aws_1.6.7`, `gce_1.7.5`, and `gce_1.8.0`?

File: tests/scripts/kubeadm-install.sh (hunk 1 above)
Comment (dangula) [reply tests/scripts/kubeadm-install.sh]: yes for K8s 1.8 we need to disable swap. I will put a comment in.

File: Jenkinsfile (hunk 2 above)
Comment (dangula) [reply Jenkinsfile]: done, using aws_1.6,gce_1.7 and gce_1.8  for stage names now.

File: tests/scripts/kubeadm-install.sh [hunk 3]
```diff
@@ -5,6 +5,8 @@ KUBE_VERSION=${1:-"v1.7.5"}
 null_str=
 KUBE_INSTALL_VERSION="${KUBE_VERSION/v/$null_str}"-00
 
+#need to disable swap when installing k8s 1.8.x via kubeadm
```
Comment (kokhang) [inline tests/scripts/kubeadm-install.sh]: You might want to provide a link to the issue. https://github.com/kubernetes/kubernetes/issues/34726

**Pull request merged**